### ISSUE TITLE
html title

### ISSUE THREAD

**Issue by xamanu** [hotosm/tasking-manager#2614]
Improve title shown in browser tabs:

![image](https://user-images.githubusercontent.com/97706/78052913-50a86a00-736f-11ea-958c-04780eb78445.png)

* [x] Make html title configurable additionally to title (may differ because of the logo). Environment variable with the default/fallback `Tasking Manager`
* [x] Include environment variable for our deployment with `HOT Tasking Manager`
* [ ] Fix the title tag when displaying iD to have the same value

**Comment by willemarcel** [comment]
With https://github.com/hotosm/tasking-manager/pull/2637, I couldn't prevent iD of setting the title tag, but I set the title tag on all project pages. So the title tag is updated when the user finished mapping on iD and go back to the task selection page.

I also added the username to the title tag when on `/users/username/`. I propose we set different title tags to each page, like, explore, about, learn, my contributions... It's not difficult to do it
